styles.css

```diff
@@ -2,6 +2,10 @@
 @import url('https://fonts.googleapis.com/css?family=Playfair Display');
 @import url('https://fonts.googleapis.com/css?family=Poppin');
 
+html {
+  overflow-x: hidden;
```

Comment:
Good job on trying to fix the horizontal scrolling issue , To solve this problem move this property to the body